airbyte-cdk/bulk/core/base/build.gradle.kts

```diff
@@ -22,7 +22,7 @@ dependencies {
     api("com.fasterxml.jackson.core:jackson-databind")
     api("com.fasterxml.jackson.datatype:jackson-datatype-jsr310")
     api("com.kjetland:mbknor-jackson-jsonschema_2.13:1.0.39")
-    api("io.airbyte.airbyte-protocol:protocol-models:0.15.0") {
+    api("io.airbyte.airbyte-protocol:protocol-models:0.16.0") {
```

Comment:
This will lead to compilation errors on when clause that map on a DestinationSyncMode and we will have to address those